src/server/routes/main.routes.js

```diff
@@ -147,5 +153,57 @@ MainRoutes.get('/pdf/:id', async function (req, res) {
   }
 });
 
+MainRoutes.get('/howru', async (req, res) => {
+  const redisInstance = req.app.get('redisInstance');
+  const communityResponse = await req.callServiceProvider('howruCommunity');
+  const adminResponse = await req.callServiceProvider('howruAdmin');
+  const openPlatformResponse = await req.callServiceProvider('howruOpenPlatform');
+  const openAgencyResponse = await req.callServiceProvider('howruOpenAgency');
+  const openUserstatusResponse = await req.callServiceProvider('howruOpenAgency');
+  const entitySuggestResponse = await req.callServiceProvider('howruEntitySuggest');
+
+  const response = {
+    ok: true,
+    services: [
+      {
+        service: 'redis',
+        ok: redisInstance.client.connected
+      },
+      {
+        service: 'CommunityService',
+        ok: communityResponse[0]
+      },
+      {
+        service: 'bibloadmin',
+        ok: adminResponse[0]
+      },
+      {
+        service: 'openlatform',
+        ok: openPlatformResponse[0][0]
+      },
+      {
+        service: 'smaug',
+        ok: openPlatformResponse[0][1]
+      },
+      {
+        service: 'openagency',
+        ok: openAgencyResponse[0]
+      },
+      {
+        service: 'openuserstatus',
+        ok: openUserstatusResponse[0]
+      },
+      {
+        service: 'entitysuggest',
+        ok: entitySuggestResponse[0]
+      }
+    ],
+    version: res.locals.gitsha,
+    env: req.app.locals.env,
+    config: filterConfig(req.app.get('BIBLO_CONFIG'))
```

Comment:
Kan der ligge informationer, som ikke må være tilgængelig for offentligheden? Eller sikrer vi på anden hvis at endpointet ikke er tilgængelig udefra?

Reply:
Ja det kan der vel godt. Tænker det er lidt jeres vurdering. Brugernavne, password mm. filtreres fra men er der f.eks. hårdt kodet bruger/password i en endpoint-streng vil det blive eksponeret. Det er der dog ikke pt.

I kunne jo evt. konfigurerer proxyen så `/howru` kun er tilgængeligt 'indefra'.